app/controllers/image_uploads_controller.rb

```diff
@@ -28,18 +26,28 @@ def create
       return
     end
 
-    cloudinary_link(uploader)
+    cloudinary_link(uploaders)
   end
 
-  def cloudinary_link(uploader)
-    link = if params[:wrap_cloudinary]
-             ApplicationController.helpers.cloud_cover_url(uploader.url)
-           else
-             uploader.url
-           end
-
+  def cloudinary_link(uploaders)
+    links = if params[:wrap_cloudinary]
+              [ApplicationController.helpers.cloud_cover_url(uploaders[0].url)]
```

Comment:
shouldn't this iterate on all uploaders?

Reply:
I don't iterate on all uploaders because `wrap_cloudinary` is passed when the cover image is uploaded. I expect it should be one cover image per article.

https://github.com/thepracticaldev/dev.to/blob/a8227b30488d00bf03ad89a987782af314f4ae35/app/javascript/article-form/elements/imageManagement.jsx#L16-L25